<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalcLog</class>
 <widget class="QMainWindow" name="CalcLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Caculator log</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="calcLog">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>241</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="openFileAction"/>
    <addaction name="saveCurrentFileAction"/>
    <addaction name="saveFileAction"/>
    <addaction name="revertFileAction"/>
   </widget>
   <widget class="QMenu" name="menuCalculator">
    <property name="title">
     <string>Calculator</string>
    </property>
    <addaction name="showCalcAction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCalculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="openFileAction">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="saveCurrentFileAction">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="saveFileAction">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="revertFileAction">
   <property name="text">
    <string>Revert</string>
   </property>
  </action>
  <action name="showCalcAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Calculator</string>
   </property>
   <property name="toolTip">
    <string>Show/hide calculator</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
